app/views/employees/_employees_detail.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="row justify-content-center">
+  <div class="col-md-10">
+    <table class="table table-bordered table-striped table-hover text-center">
+      <tr>
+        <th>Id</th>
+        <th>Name</th>
+        <th>Email</th>
+        <th>Gender</th>
+        <th>Hobbies</th>
+        <th>Address</th>
+        <th>Mobile Number</th>
+        <th>Birthdate</th>
+        <th>Uploaded File</th>
+        <% if controller.action_name == "index" %><th>Action</th><% end %>
```

Comment:
Your previous condition was correct, However, I'm asking for a reason to add this condition.